samples/04_gis_analysts_data_scientists/building_reconstruction_using_mask_rcnn.ipynb

```diff
@@ -195,22 +195,25 @@
   },
```

Comment:
<p>Should the connection be put back to 'home'?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/789/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>